### ISSUE TITLE
[ErrorHandling] Blank page instead of error message

### ISSUE THREAD

**Issue by kevou84** [symfony/symfony#14669]
Hi guys,

**On Synfony 2.8.x**

I had a blank page instead of an error when calling a *deprecated* function in app/autoload.php :
`$loader->registerPrefixes(...);`

NB : For those who encounter a blank page once in their lifetime, launching a command line such as "php app/console cache:clear --env=prod --no-debug" will generally detect and display the hidden error (in this case it helped ^^).

Cheers,

**Comment by Tobion** [comment]
Did you do an upgrade and cleared the cache?

**Comment by Kevou** [comment]
Hi Tobias,

If by an upgrade you mean `php composer.phar update `, I did it 1 hour before modifying autoload.php and making this error. So I was at last commit of 2.8.x-dev. I deleted the dev folder, but blank page stayed, then launched the command line I mention in my post and it displayed the error in my cmd window. When I solved the error, no more blank page.

Cheers,

**Comment by nicolas-grekas** [comment]
What was the error? Only the deprecation of something else? When in this erroneous situation, does your `app/bootstrap.php.cache` start with:
```php
namespace {
    error_reporting(error_reporting() & ~E_USER_DEPRECATED);
    $loader = require_once __DIR__.'/autoload.php';
}
```
?

**Comment by Kevou** [comment]
Hi Nicolas, sorry I used the term *deprecation* because i feel based on my experience (and examples found on internet) that $loader->registerPrefixes() used to work well. The error message, however, did not indicate a deprecation but simply a call to unexisting function.

I replaced $loader->registerPrefixes() by $loader->add() after a research of my own and it works well. I was just pointing the blank page phenomenon upon error. For the record my boostrap file is normal : 

````
namespace {
    error_reporting(error_reporting() & ~E_USER_DEPRECATED);
    $loader = require_once __DIR__.'/./autoload.php';
}
````
